s3-proxy/nginx.conf

```diff
@@ -69,6 +69,11 @@ http {
             add_header 'Content-Type' 'text/plain';
             return 200 'S3 Proxy';
         }
+
+        location = /robots.txt {
```

Comment:
Does this work for head requests? Some bots HEAD before GET, etc.

Reply:
Yep